### DISCUSSION TITLE
Farm Summary - Fixing "No Plots"

### DISCUSSION THREAD

**Discussion by KnowTheFuture** [Chia-Network/chia-blockchain#12927] [Support]
Hi all,

My NAS was down for a few days after upgrading the Docker container from 1.3.4 to 1.5.0

Before upgrading, all of the plots were set up and everything was working well. The settings are the same in 1.5.0. 

"Plots Show" lists the directories and "Plots Check" confirms there are about 900 plots.
<img width="963" alt="Screen Shot 2022-08-11 at 21 52 34" src="https://user-images.githubusercontent.com/8218649/184307880-1f59f62b-bd9b-4e43-88a9-7766480c1bf9.png">

However, "Farm Summary" says "No plots"
<img width="632" alt="Screen Shot 2022-08-12 at 16 18 53" src="https://user-images.githubusercontent.com/8218649/184308023-a73e7931-592d-4d0b-a604-7c27c921b9b1.png">

Running the same set-up on my plotting machine with a few external HDD's connected and the information for the local machine is shown immediately.
<img width="682" alt="Screen Shot 2022-08-12 at 16 38 22" src="https://user-images.githubusercontent.com/8218649/184308090-c7f608aa-1d30-40ca-8922-4a0025e3e8d7.png">

 Any suggestions?

**Comment by mouracomp2** [comment]
<html>
<body>
<div class="aju" style="cursor: default; float: none; padding: 0px 16px; align-items: center; display: flex; height: 80px; min-width: 40px;"><div class="aCi" style="position: relative;"><img id=":177_10-e" name=":177" src="https://ssl.gstatic.com/ui/v1/icons/mail/profile_mask2.png" class="ajn" jid="mouracomp@gmail.com" data-hovercard-id="mouracomp@gmail.com" data-name="Carlos Moura" aria-hidden="true" style="display: block; width: 40px; height: 40px; border-radius: 50%; background-color: rgb(204, 204, 204);"></div></div><div class="gs" style="margin: 0px; padding: 0px 0px 20px; width: 950px;"><div class="gE iv gt" style="font-size: 0.875rem; padding: 20px 0px 0px; cursor: auto;">


</div><div id=":2st"><div class="qQVYZb"></div><div class="utdU2e"></div><div class="lQs8Hd" jsaction="SN3rtf:rcuQ6b" jscontroller="i3Ohde"></div><div class="btm"></div></div><div class=""><div class="aHl" style="margin-left: -38px;"></div><div id=":2r0" tabindex="-1"></div><div id=":3j0" class="ii gt" jslog="20277; u014N:xr6bB; 4:W251bGwsbnVsbCxbXV0." style="direction: ltr; margin: 8px 0px 0px; padding: 0px; position: relative; font-size: 0.875rem;"><div id=":3iz" class="a3s aiL " style="font: small / 1.5 Arial, Helvetica, sans-serif; overflow: hidden;"><div dir="ltr">I come through this to obtain information, from blocks collected in SpacePool, because my percentage in my wallet is not paid.<div>my Wallet 280308653<br></div><div>my payment address was<br>xch1u4tycqne2aqrgcvvfnk2sjqlgw<wbr>ynucqggv6j4kg97wps3hkj4wds5kn7<wbr>3p<br></div><div>Date<br>08/08/2022 00:21</div><div>07/23/2022 02:07<br>04/25/2022 06:25<br>03/22/2022 14:01<br>12/29/2021 20:12<br>block #<br>2369220<br>2295897<br>1886491<br>1730969<br>1349021<br>hash<br>08795929d431ec810371dfe48e609e<wbr>b1d1e395a96be78186627c178ad6d9<wbr>b742</div><div><br>6896a896874b87b5be2743ce61a53c<wbr>0ea3ce759c30b81c393eed25f9149b<wbr>1af5</div><div><br>57304513456528bd42bcf36785ff47<wbr>ba9670c971dc949658def3e2a610ef<wbr>3fa0</div><div><br>ff2f214133248308b1b4459dd9f296<wbr>da1a7430554919185713e2b3b9522c<wbr>f2ee</div><div><br>29c941f2d9dfe4a4e687bbc6111903<wbr>3e5249f227d7a994d4f91533f7b174<wbr>f420<br></div><div><br></div><div>never been paid, any explanation, what am i doing wrong?<div class="yj6qo"></div><div class="adL"><br style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
</body>
</html>
